pxc/jenkins/pxc80-pipeline.groovy

```diff
@@ -142,10 +146,40 @@ pipeline {
                     fi
                     rm -f ${WORKSPACE}/VERSION-${BUILD_NUMBER}
                 '''
+                sh '''
+                echo 'Getting percona-xtrabackup repo'
+                if [ -f /usr/bin/yum ]; then
+                    sudo yum -y install git
+                else
+                    sudo apt-get install -y git
```

Comment:
is apt cache up to date on this stage?

Reply:
I have added a command at the start of the current step.